addons/website_forum/views/website_forum.xml

```diff
@@ -671,13 +951,15 @@
     <t t-set="karma_down" t-value="forum.karma_downvote"/>
     <t t-set="can_upvote" t-value="post.can_upvote"/>
     <t t-set="can_downvote" t-value="post.can_downvote"/>
-    <div class="vote text-center">
-        <a t-attf-class="vote_up fa fa-caret-up no-decoration #{own_vote == 1 and 'text-success' or ''} #{((own_vote == 1 and not can_downvote) or not can_upvote) and 'karma_required' or ''}"
+    <div t-attf-class="vote text-center d-inline-flex align-items-center #{vertical and 'o_wforum_vote_vertical flex-md-column'} #{classes}">
+        <a t-attf-class="btn vote_up fa fa-caret-up pl-0 #{'px-md-0 pt-md-0' if vertical else 'pr-2'} #{own_vote == 1 and 'disabled text-success'} #{((own_vote == 1 and not can_downvote) or not can_upvote) and 'karma_required'}"
+            data-active-class="text-success"
             t-attf-data-karma="#{own_vote == 1 and karma_down or karma_up}"
```

Comment:
The display/animation of the feature appears broken since at least 12.0 but before your refactoring, someone was able to remove its upvote or downvote (by either clicking on the same button or click on the other button). I think this should stay, if I click by mistake it seems to be legit to remove the vote don't you think ? (and otherwise, all the code here is dead...)

Reply:
>  but before your refactoring, someone was able to remove its upvote or downvote (by either clicking on the same button or click on the other button). 

Sure, but It should still be the case actually. I'm testing the branch, I can upvote and then downvote. The following code should not be dead, it sets the initial state (if you previously upvoted, downvoted). I'm probably missing something

Reply:
No, you are marking the button as disabled and the behavior is broken when clicking on the other button.

Simple flow:
- The score is currently 9
- I hit "up vote" by mistake -> the score is now 10 and it's green
- I want to remove by reclicking, I can't
- I try hitting the "down vote" instead -> the score is now back to 9... but it's red ?
- I am not able to "down vote" to 8 since it is now disabled
- I reload the page and it is still 9 but not red anymore

I refactored the JS/XML in my branch but the behavior is still the same... what would be the behavior you wanted? ;)

Reply:
Mmh, I guess if reddit does this... I suppose our original behavior is the one of stackoverflow and it feels better to me :/ I would personally never dare hitting the down vote button to undo my up vote... Anyways, I'll fix this in my branch now to match what you desired.

Reply:
In fact, I will push my refactoring with the broken behavior today and speak with @JKE-be tomorrow about this. The code seems too broken in stable version to me to think about it x) Reading the code, it seems that you could remove your upvote directly if not reloading the page and may not have enough karma to do it after reloading the page

Reply:
@JKE-be Still need to talk about this in fact... :/